kratos/geometries/geometry.h

```diff
@@ -676,6 +701,70 @@ class Geometry
             << std::endl;
     }
 
+    ///@}
+    ///@name Geometry part functions
+    ///@{
+
+    /**
+    * @brief Used for composite geometries. It returns the
+    *        the geometry part, corresponding to the Index.
+    * @param Index of the geometry part. This index can be used differently
+    *        within the derived classes.
+    * @return reference to corresponding geometry.
+     */
+    virtual GeometryType& GetGeometryPart(IndexType Index) const
```

Comment:
Should here also exist the const GetGeometryPart() const; plus GetGeometryPart()?
@philbucher @pooyan-dadvand